time t
string mac
string ssid
int32 signal_level
int32 signal_noise
